apps/consumer-server/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "include": ["**/*.ts", "**/*.d.ts"],
   "exclude": ["node_modules"],
   "compilerOptions": {
+    "types": ["mocha"],
```

Comment:
I suggest adding inline documentation here and in the 2nd file of this PR that
1) links to this PR
2) briefly summarizes the reason we need this